tests/std/tests/GH_000519_cmath_overloads/test.cpp

```diff
@@ -158,7 +159,94 @@ CONSTEXPR23 void test_other_integral_overloads() {
     }
 }
 
-CONSTEXPR23 bool test_all_integral_overloads() {
+template <class F>
+CONSTEXPR23 void test_floating_overloads() {
+    constexpr F zero = 0;

```

Comment:
Here's hoping Nintendo lawyers don't come after us for using `F zero` in our sources =)